docs/manage-metadata-using-gravitino.md

```diff
@@ -961,9 +964,11 @@ You can remove a table by sending a `DELETE` request to the `/api/metalakes/{met
 <TabItem value="bash" label="Bash">
 
 ```bash
+## purge can be true or false, if purge is true, Gravitino will remove the real data of the table.
```

Comment:
I think you can remove "real", it not necessary to emphasize it.

Reply:
Fixed

Reply:
Did you fix it?

Reply:
Oh, I removed the data in line 992. Line 967 should also do it.

Reply:
done